config/sprint/label_templates/plate_384.yml.erb

```diff
@@ -0,0 +1,48 @@
+[
```

Comment:
Do we want to commit these ymls? The real ones will come from the psd-deployment project right?